docs/concepts.rst

```diff
@@ -97,6 +97,8 @@ any of its operators. This makes it easy to apply a common parameter to many ope
     op = DummyOperator(task_id='dummy', dag=dag)
     print(op.owner) # Airflow
 
+.. _default-args:
```

Comment:
This doesn't quite make sense as the target for the link (it applies to the section after it. I think you want to move this to before the `Default Arguments` heading.